packages/netskope/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml

```
@@ -11,7 +11,7 @@ processors:
       on_failure:
         - append:
             field: error.message
-            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
+            value: '{{{_ingest.on_failure_message}}}'
```

I think this is a DX regression.

---

I agree. That conveys less context. 

I copy/paste from this [here](https://github.com/andrewkroh/go-examples/blob/791dbc2e05bd9fb0f7a10a888d6a0669019a867b/ecs-update/ecs-update.go#L1444-L1448) on occasion. It conditionally inserts the processor's `tag` into the error if it exists.

```suggestion
            value: >-
              Processor '{{ _ingest.on_failure_processor_type }}'
              {{#_ingest.on_failure_processor_tag}}with tag '{{ _ingest.on_failure_processor_tag }}'
              {{/_ingest.on_failure_processor_tag}}failed with message '{{ _ingest.on_failure_message }}'
```